### PR TITLE
Enables force-destroy of S3 origin bucket

### PR DESCRIPTION
**Pull request by Moondancer83**
Adding option to set up force-destroy on origin S3 bucket.

### CODE REVIEW

**Review by iuriaranda — APPROVED** [review]

**Comment by iuriaranda** [comment]
Thanks for the contribution. Waiting for another approval from the team and then I'll merge the PR.

**Review by duboisph — APPROVED** [review]

**Pull request merged**
